config/OPM-Reconfig.cfg

```diff
@@ -0,0 +1,499 @@
+// Original draft 2019/02/20 by Eridan/HSJasperism
+
```

Comment:
Please add a comment saying what versions this config is for. I think in this case those would be Custom Asteroids 1.6+ (to support localization tags) and Outer Planets Mod 2.0+ (unless it depends on more recent changes to the Kopernicus config?).

Reply:
Added comment for CA 1.6.
With regards to OPM, Kopernicus Asteroids were adopted with 2.2.2 (the latest release). This patch is targeting that build.

Kopernicus Asteroids tends to spawn bizarrely when multiple Kopernicus { Asteroid {} }  config nodes are active. It isn't obvious on install since the included configs are off until reached. I usually only get the last loaded Asteroid {} set to spawn with proper probabilities, and anything loaded earlier will barely spawn anything. A few questions have been raised on the OPM thread, but this is mostly a Kopernicus issue. Modifying the default Asteroid {} config node is an option, but it makes balancing probabilities a pain for each individual planet pack installed (e.g. Stock + GPP systems, galaxy mods, etc).